xskillscore/core/test.py

```diff
@@ -0,0 +1,110 @@
+import scipy.stats as st
+import xarray as xr
+
+
+def sign_test(
+    forecast1, forecast2, observation, dim=None, categorical=False, alpha=0.05
+):
+    """
+        Returns the Delsole and Tippett sign test over the given time dimension.
+
+        Parameters
+        ----------
+        forecast1 : xarray.Dataset or xarray.DataArray
+            forecast1 to be compared to observation
+        forecast2 : xarray.Dataset or xarray.DataArray
+            forecast2 to be compared to observation
+        observation : xarray.Dataset or xarray.DataArray
+            observation to be compared to both forecasts
+
+            if str, then assume that forecast1 and forecast2 have already been compared
+            to observations and choose from:
+
+                * ``negatively_oriented_already_evaluated``: metric between forecast1
+                  (forecast2) and observations. Distances are positively oriented,
+                  therefore the smaller distance wins.
+                * ``positively_oriented_already_evaluated``: metric between forecast1
+                  (forecast2) and observations. The larger positively oriented metric
+                  wins.
+                * ``categorical_already_evaluated``: categorical data following
+                  ``logical(forecast1)==logical(forecast2)`` where ``logical`` is a
```

Comment:
I assume this is supposed to be `logical(forecast#)==logical(observation)`? But, even still, I'm struggling to interpret this. Categorical data need not be binary. It should be N integers corresponding to a set of N categories.

Reply:
I thought only about a binary `True/False`. thanks for the feedback

Reply:
this docstring is gone and I dont touch the categorical part. think this is fine as it is now